### PR TITLE
Modified USER models

### PR DESCRIPTION
**Pull request by Abduraxmonnn**
changed imports and USER models.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -132,3 +132,4 @@ files
 .coveragerc
 coverage_test.sh
 /htmlcov
+/apps/main/
```
Comment (ChogirmaliYigit) [inline .gitignore:135]: Why are you ignoring the `main` app?

File: apps/users/models.py [hunk 2]
```diff
@@ -27,4 +28,4 @@ def full_name(self):
         return full_name.strip()
 
     class Meta:
-        db_table = "users"
+        db_table = "user"
```
Comment (ChogirmaliYigit) [inline apps/users/models.py]: we may have a lot of users, so fix the naming to `users`

File: config/settings.py [hunk 3]
```diff
@@ -40,8 +40,9 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "core",
-    "users",
+    "apps.core",
+    "apps.users",
+    # "apps.main.mentors"  # This app is in .gitignore
```
Comment (ChogirmaliYigit) [inline config/settings.py]: If the app should be ignored, please don't write it like that. Just remove your comments

File: .gitignore (hunk 1 above)
Comment (Abduraxmonnn) [reply .gitignore:135]: > Why are you ignoring the `main` app?

because somone should create files structure.

File: apps/users/models.py [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
 
 class User(BaseModel, AbstractBaseUser, PermissionsMixin):
-    first_name = models.CharField(max_length=1000, null=True, blank=True)
-    last_name = models.CharField(max_length=1000, null=True, blank=True)
-    username = models.CharField(max_length=1000, unique=True)
-    email = models.EmailField(unique=True)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255, null=True, blank=True)
+    phone_number = models.CharField(max_length=50)
+    age = models.IntegerField()
```
Comment (ChogirmaliYigit) [inline apps/users/models.py]: `age` should be `PositiveIntegerField()`

**Review by ChogirmaliYigit — COMMENTED** [review]
Fix by my comments

File: apps/users/models.py (hunk 4 above)
Comment (Abduraxmonnn) [reply apps/users/models.py]: what difference between ? anyway frontend or backend can check this field, if needed.

**Pull request closed without merging**